src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -600,10 +600,6 @@ public function mirror($originDir, $targetDir, \Traversable $iterator = null, $o
      */
     public function isAbsolutePath($file)
     {
-        if (null === $file) {
-            @trigger_error(sprintf('Calling "%s()" with a null in the $file argument is deprecated since Symfony 4.4.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Should we throw instead?

Reply:
There is another PR adding a type hint for this argument

Reply:
#32247